apps/docs/src/css/custom.css

```diff
@@ -176,7 +176,9 @@
  }
 
  hr {
-   @apply h-[0.5px] my-5 border-border !important;
+   @apply h-[1px] border-border !important;
```

Comment:
Do we need this? 👁️

Reply:
it's just a visual improvement for the doc 😬